libmultilabel/linear/preprocessor.py

```diff
@@ -51,6 +56,7 @@ def load_data(
             label_file (str, optional): Path to a file holding all labels. Defaults to None.
             include_test_labels (bool, optional): Whether to include labels in the test dataset. Defaults to False.
             remove_no_label_data (bool, optional): Whether to remove training instances that have no labels. Defaults to False.
+            return_raw: Whether to return raw libmultilabel format data.
 
         Returns:
             dict[str, dict[str, sparse.csr_matrix]]: The training and test data, with keys 'train' and 'test' respectively.
```

Comment:
descriptions here should be modified as the returned object may not be a sparse csr matrix?

Reply:
The returned type has been corrected